### ISSUE TITLE
codeceptjs 2.0: "Could not load helper Appium ..." though webdriverio@5 is installed

### ISSUE THREAD

**Issue by hubidu** [Codeception/CodeceptJS#1390]
#### What are you trying to achieve?

After upgrading to codeceptjs 2.0 I want to run my appium tests

#### What do you get instead?

```bash
Could not load helper Appium from module './helper/Appium':
Required modules are not installed.

RUN: npm install --save-dev webdriverio@5
```

However webdriverio@5 is already installed and the real underlying cause seems to be this error

```bash

Caused By: Error: Cannot find module 'source-map-support/register'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\stefan.huber\projects\codeceptjs-mobile\node_modules\webdriverio\build\index.js:8:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
  nested: { Error: Cannot find module 'source-map-support/register'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\stefan.huber\projects\codeceptjs-mobile\node_modules\webdriverio\build\index.js:8:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3) code: 'MODULE_NOT_FOUND'

```

which is thrown when trying to load webdriverio

### Details

* CodeceptJS version: v2.0.0
* NodeJS Version: v8.11.3
* Operating System: Windows/Linux
* Appium
* Configuration file:

```js

{
  "name": "versicherungscenter-android",
  "tests": "./**/*_test.js",
  "timeout": 10000,
  "output": "./output",
  "helpers": {
    "Appium": {
      "host": "xxx.xxx.xxx.xxx",
      "port": 4723,
      "waitForTimeout": 30000,
      "app": "c:\\Temp\\check24.apk",
      "desiredCapabilities": {
        "platformName": "Android",
        "platformVersion": "8.1",
        "deviceName": "Nexus 6 API 27",
        "automationName": "UiAutomator2",
        "nativeWebScreenshot": true
      }
    }
  },
  "include": {
    "I": "./steps_file.js"
  },
  "bootstrap": false,
  "mocha": {}
}

```

**Comment by xMutaGenx** [comment]
I'm having the same issue with loading `WebDriver` helper, doesn't seem to find wdio5, works fine if reverted to `WebDriverIO` helper with wdio4

**Comment by PeterNgTr** [comment]
Fixed here https://github.com/Codeception/CodeceptJS/pull/1396

**Comment by DavertMik** [comment]
Yeah, got the same. This was issue of webdriverio. I hope it was fixed in 2.0.1
